crates/trigger/src/cli.rs

```diff
@@ -291,6 +291,7 @@ pub mod help {
         type RuntimeData = ();
         type TriggerConfig = ();
         type RunConfig = NoArgs;
+        type InstancePre = spin_core::InstancePre<()>;
```

Comment:
Seems like this is the more common pattern?

```suggestion
        type InstancePre = spin_core::InstancePre<Self::RuntimeData>;
```